scripts/get_failed_registrations.py

```diff
@@ -0,0 +1,82 @@
+
+# -*- coding: utf-8 -*-
+import sys
+import logging
+
+from datetime import datetime
+from django.core.mail import send_mail
```

Comment:
Use the OSF's `send_mail` function rather than Django's. Otherwise, this script will likely crash.